src/components/Modal/Modal.spec.tsx

```diff
@@ -15,18 +15,21 @@
 
 import React from 'react';
 
-import { ModalConsumer, ModalProvider } from '.';
+import { render, act, userEvent, wait } from '../../util/test-utils';
```

Comment:
Why is it named `userEvent`? Aren't we using rtl for this?

Reply:
`@testing-library/user-event` is part of the RTL ecosystem:

> `user-event` is a companion library for React Testing Library that provides more advanced simulation of browser interactions than the built-in `fireEvent` method.

From https://testing-library.com/docs/ecosystem-user-event

Reply:
I just saw from this blogpost: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library. Looks like they are adding lot's of useful goodies (like find*). Good to know!